src/components/editProfile/EditForm.tsx

```diff
@@ -0,0 +1,54 @@
+import CbrumIcon from 'components/common/inside/cbrum/cbrumIcon'
+import TextInput from 'components/common/inside/forms/textInput'
+import editProfileService from 'services/users/editProfileService';
+
+import 'components/editProfile/EditForm.css'
+import rightArrow from 'assets/images/arrow_left.png'
+import { useState } from 'react';
+
+const editProfileFunction = (email: string, password: string, newPassword: string, confirmNewPassword: string) => {
+  const res = editProfileService(email, password, newPassword, confirmNewPassword);
+
+  res instanceof Error? new Notification("Profile NOT updated!") : new Notification("Profile updated!");
+}
+
+const EditForm = () => {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [newPassword, setNewPassword] = useState<string>("");
+  const [confirmNewPassword, setConfirmNewPassword] = useState<string>("");
+
+  return(
+    <div className='panel-body-form'>
+      <div className='target--header'>
+        <img src={rightArrow} className='target--left-arrow' />
+        <div className='target--header-title'>
+          edit profile
+        </div>
+      </div>
+      <CbrumIcon />
+      <div className='panel-mid-section--body'>
+        <form
+          onSubmit={e => {
+            e.preventDefault();
+            editProfileFunction(email, password, newPassword, confirmNewPassword);
+          }}
```

Comment:
This is not blocking at all, but it's a good pratice to actually pass an object as an argument with properties instead of individual arguments. This way, another programmer will understand easier what does your function do and what arguments requires.
In this example, your arguments are self-explanatory but in the future you can do something like this
```suggestion
            editProfileFunction({email, password, newPassword});
          }}
```

Reply:
Oh right, I'll keep that in mind